### PR TITLE
fix Implicit memory aliasing in for loop

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go [hunk 1]
```diff
@@ -178,18 +178,18 @@ func (r *managedReconcile) cleanUpAppliedManifestWorks(ctx context.Context, klus
 	}
 
 	var errs []error
-	for _, appliedManifestWork := range appliedManifestWorks.Items {
+	for index, appliedManifestWork := range appliedManifestWorks.Items {
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go]: should this be `for index := range appliedManifestWorks.Items`?

Comment (ldpliu) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go]: > should this be `for index := range appliedManifestWorks.Items`?

done

**Pull request merged**